src/app_charts/prometheus/prometheus-robot.values.yaml

```diff
@@ -88,14 +88,23 @@ prometheusOperator:
 kubeApiServer:
   serviceMonitor:
     interval: 10m
+    metricRelabelings:
+    # Drop high cardinality apiserver buckets.
+      - action: drop
+        regex: "^apiserver_request|etcd_request"
```

Comment:
nit: `^apiserver_request|^etcd_request`, otherwise this will also match `not_an_etcd_request`. Unlikely to ever affect anything but I guess it's not what you intended.